### PR TITLE
WIP: reclassify terms for peroxisome biogenesis disorder

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
address #2632

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
to do:
- [ ] add clingen_preferred tag to clingen terms
- [ ] add disease_grouping tag to grouping classes like: 'peroxisome biogenesis disorder due to PEX1 defect'

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -166528,7 +166530,7 @@ xref: UMLS:C0043459 {source="MONDO:subClassOf", source="OMIM:214100", source="NC
 xref: UMLS:C1865803 {source="OMIM:214100", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C1865804 {source="OMIM:214100", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0019234 {source="MONDO:0008953/inferred", source="MONDO:Redundant", source="MONDOLEX:0008953/inferred", source="OMIM:214100", source="indirect"} ! peroxisome biogenesis disorder
-is_a: MONDO:0019609 {source="DC-OMIM:214100", source="MONDOLEX:0008953", source="ORDO-Gene"} ! Zellweger syndrome
+is_a: MONDO:0100259 {source="DC-OMIM:214100", source="MONDOLEX:0008953", source="ORDO-Gene", source="https://github.com/monarch-initiative/mondo/issues/2632", source="https://www.clinicalgenome.org/affiliation/40049/"} ! peroxisome biogenesis disorder due to PEX1 defect
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:166533]: If the subclass axiom changed, shouldn't the sources be removed? Are these "sources" now really the sources of the subsumption? Also, what is `DC-OMIM:`?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -166544,7 +166546,7 @@ xref: UMLS:C1859228 {source="NCBI:mim2gene_medline", source="MONDO:notFoundInDis
 xref: UMLS:C3550273 {source="OMIM:214110", source="MONDO:equivalentTo"}
 xref: UMLS:C3550274 {source="OMIM:214110", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0019234 {source="MONDO:0008954/inferred", source="MONDO:Redundant", source="MONDOLEX:0008954/inferred", source="OMIM:214110", source="indirect"} ! peroxisome biogenesis disorder
-is_a: MONDO:0019609 {source="DC-OMIM:214110", source="MONDOLEX:0008954", source="ORDO-Gene"} ! Zellweger syndrome
+is_a: MONDO:0100262 {source="DC-OMIM:214110", source="MONDOLEX:0008954", source="ORDO-Gene", source="https://github.com/monarch-initiative/mondo/issues/2632", source="https://www.clinicalgenome.org/affiliation/40049/"} ! peroxisome biogenesis disorder due to PEX5 defect
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:166549]: Same as above, should the sources be changed? what is `DC-OMIM`?

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -167008,7 +167010,7 @@ xref: Orphanet:177 {source="MONDO:subClassOf", source="OMIM:215100"}
 xref: Orphanet:309789 {source="OMIM:215100", source="MONDO:equivalentTo", source="DOID:0110851"}
 xref: UMLS:C1859133 {source="OMIM:215100", source="NCBI:mim2gene_medline", source="MONDO:equivalentTo", source="ORDO:309789/e", source="Orphanet:309789"}
 is_a: MONDO:0015776 {source="DOID:0110851", source="MONDO:Redundant", source="MONDOLEX:0008972", source="OMIM:215100", source="Orphanet:309789"} ! rhizomelic chondrodysplasia punctata
-is_a: MONDO:0019234 {source="MONDOLEX:0008972", source="OMIM:215100"} ! peroxisome biogenesis disorder
+is_a: MONDO:0100272 {source="https://github.com/monarch-initiative/mondo/issues/2632", source="https://www.clinicalgenome.org/affiliation/40049/"} ! peroxisome biogenesis disorder due to PEX7 defect
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:167013]: seems odd to have an issue on github be the source of a subsumption.. Isnt this a rdfs:seeAlso? I don't know that but whatever it is, it should be standard.

Comment (matentzn) [reply src/ontology/mondo-edit.obo:167013]: Happens multiple times throughout PR, just wondering if that is the accepted practice.

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -233095,11 +233109,21 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/10
 
 [Term]
 id: MONDO:0011905
-name: obsolete familial hypercholanemia
-property_value: IAO:0000231 IAO:0000229
-property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2940 xsd:string
-is_obsolete: true
-replaced_by: MONDO:0031446
+name: familial hypercholanemia
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:233112]: This is odd - a previously obsolete class gets re-instated? Afaik this should not really be done, but if that is common practice...

**Review by matentzn — COMMENTED** [review]
I left some comments for you @nicolevasilevsky to double check. I would spent like 20 min scrolling through this PR to see whether its all in order!

**Comment by matentzn** [comment]
There is also a proxy merge that needs fixing.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:166533]: DC-OMIM is disease cluster OMIM. It was something we were using before Mondo was created. Chris has retained these IDs for historical purposes.

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:233112]: no, this should not be reinstated. That is the problem with this PR, it is reinstated for some reason and it shouldn't be.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: I re-obsoleted this (I looked at the file on master and copied and pasted the lines into this PR)

**Pull request merged**